GG/GG/Font.h

```diff
@@ -206,6 +209,22 @@ class GG_API Font
 
         virtual ~TextElement(); ///< Virtual dtor.
 
+        /** Rebind to \p whole_text, by binding the SubString data member text to \p whole_text.
+
+            Binding to a new \p whole_text is very fast compared to re-parsing the entire \p
+            whole_text and allows TextElements of TextElementType TEXT to be changed quickly if it
+            is known that the parse would be the same.
+
+            One use case is copying or copy constructing a TextControl.
+
+            Another use case is constructing a string and TextElement vector pair with
```

Comment:
I prioritized working on #1083 to fix the combat log glitches introduced by the merge of #1076, since that is a problem in master.

I'll try to fix this comment again.

I'll point out again that the `Substring` design is the original implementation.  I assume that it was profiled when it was designed and that creating many `Substrings` was either an unacceptable time and/or memory overhead.